components/workspaces/base/server/config/default/kustomization.yaml

```diff
@@ -25,31 +25,13 @@ replacements:
     select:
       group: rbac.authorization.k8s.io
       kind: RoleBinding
-      name: rest-api-server:spacebinding-reader
```

Comment:
I'd prefer to have the resource on which we are going to apply replacements to be explicitly declared here instead of having kustomize apply replacements on all the Role/RoleBinding. Is there any specific reason other than conciseness for this change?

Reply:
Not really, I just found it repetitive to have the exact same replacement listed multiple times.  I figured it would make sense to get kustomize to do this for us.